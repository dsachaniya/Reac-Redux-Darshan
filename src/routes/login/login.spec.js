import React from 'react';
import { shallow } from 'enzyme';
import Login from './login';

jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    push: jest.fn(),
  }),
}));
describe('<Login />', () => {
  it('Rendering', () => {
    const wrapper = shallow(<Login />);
    expect(wrapper).toMatchSnapshot();
  });
});
