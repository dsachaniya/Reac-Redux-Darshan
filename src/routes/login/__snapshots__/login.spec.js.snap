// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> Rendering 1`] = `
<section>
  <WithStyles(ForwardRef(Container))
    component="main"
    maxWidth="xs"
  >
    <h1>
      Login Page
    </h1>
    <InputBox
      autoFocus={false}
      helperText=""
      id="username_input"
      onBlur={[Function]}
      onChange={[Function]}
      type="Username"
    />
    <InputBox
      autoFocus={false}
      helperText=""
      id="password_input"
      onBlur={[Function]}
      onChange={[Function]}
      type="Password"
    />
    <WithStyles(ForwardRef(Button))
      className="makeStyles-submit-1"
      color="primary"
      disabled={false}
      fullWidth={true}
      onClick={[Function]}
      variant="contained"
    >
      Login
    </WithStyles(ForwardRef(Button))>
  </WithStyles(ForwardRef(Container))>
  <WithStyles(ForwardRef(Backdrop))
    className="makeStyles-backdrop-2"
  >
    <WithStyles(ForwardRef(CircularProgress))
      color="inherit"
    />
  </WithStyles(ForwardRef(Backdrop))>
</section>
`;
